### ISSUE TITLE
new term: Response to short acting pulmonary vasodilator plus new subclasses

### ISSUE THREAD

**Issue by lcdaugherty** [obophenotype/human-phenotype-ontology#950]
After discussion with Peter and Melissa last week at the HPO workshop I have now broken down the bulk request #914 into separate requests/related (get ready!). I hope that makes things easier for you at HPO HQ. Keep up the good work!
Best regards
Louise

relates to #914 BRIDGE_PAH_HPO_terms_20160616.xlsx

1) New term: Response to short acting pulmonary vasodilator
New definition:  A short-acting vasoactive agent such as adenosine, epoprostenol, or inhaled nitric oxide.
HPO_ID superclass (parent): Abnormality of the pulmonary vasculature HP:0004930

1A) New term : No vasodilator response
New definition: No fall in mean pulmonary arterial pressure (mPAP) falls by ≥10 mmHg to an absolute value <40 mmHg without a degradation in cardiac output (CO) in response to a short-acting vasoactive agent such as adenosine, epoprostenol, or inhaled nitric oxide.
HPO_ID superclass (new parent term): Response to short acting pulmonary vasodilator

1B) New term : Acute vasodilator response
New definition: Mean pulmonary arterial pressure (mPAP) falls by ≥10 mmHg to an absolute value <40 mmHg without a decrease in cardiac output (CO) in response to a short-acting vasoactive agent such as adenosine, epoprostenol, or inhaled nitric oxide.
HPO_ID superclass (new parent term): Response to short acting pulmonary vasodilator

**Comment by pnrobinson** [comment]
Added a term "Abnormal response to short acting pulmonary vasodilator" as a child of "Functional respiratory abnormality" (the term "Abnormality of the pulmonary vasculature" is meant to be morphological"). 
I used the definition: Pulmonary vasodilator testing is performed during right-heart catheterization and involves a short-acting vasoactive agent such as adenosine, epoprostenol, or inhaled nitric oxide. The current definition of a normal (positive) response is a drop in mean pulmonary artery pressure of at least 10 mm Hg (or 20 percent) to below 40 mm Hg.
